lldb/include/lldb/API/SBProcess.h

```diff
@@ -159,6 +159,7 @@ class LLDB_API SBProcess {
   lldb::SBError Destroy();
 
   lldb::SBError Continue();
+  lldb::SBError Continue(RunDirection direction);
```

Comment:
(Probably not in scope of this review, but rather for a follow-up commit)

An API which allows to check if the given process / target supports reverse-debugging would be useful.

E.g., VS-Code support UI elements for "reverse continue" and "step back", as shown in the following screenshot

<img width="461" alt="image" src="https://github.com/user-attachments/assets/5bfe53ea-7d10-467a-a1a4-0dff93570306">

With a `bool SupportsReverseDebugging()` function, we could hide/show those buttons in VS-Code accordingly by setting / unsetting the `supportsStepBack` [capability](https://microsoft.github.io/debug-adapter-protocol/specification#Types_Capabilities)

Reply:
Yes, we can easily add that feature later.